src/components/ModalFeedback.tsx

```diff
@@ -78,19 +84,19 @@ const ModalFeedback = () => {
             title='Send'
             active={true}
             isLoading={isSubmitting}
-            isDisabled={isDisabled}
+            disabled={isDisabled}
             onClick={() => onSubmit({ close })}
           />
-          <div key='cancel' style={{ fontSize: 22, marginTop: 10, opacity: 0.5 }}>
-            <a id='skip-tutorial' onClick={() => close()}>
+          <div key='cancel' tw='mt-2.5'>
+            <TextLink onClick={() => close()} colorVariant='gray'>
               Cancel
-            </a>
+            </TextLink>
           </div>
         </div>
       )}
     >
-      <p style={{ fontSize: 18, marginBottom: 30 }}>Send us your bugs, hopes, and dreams!</p>
-      <textarea
+      <p tw='text-lg mb-5 text-center'>Send us your bugs, hopes, and dreams!</p>
```

Comment:
For my own understanding, is this the same as `` tw.p`text-lg mb-5 text-center` ``? And the same as ``style={tw`text-lg mb-5 text-center` ``?

If so, there appear to be three different syntaxes for some tailwind attributes. I'd like to better understand any reasons there may be to use more than one.